WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -31,7 +31,7 @@ enum FeatureFlag: Int, CaseIterable {
         case .unifiedAuth:
             return BuildConfiguration.current ~= [.localDeveloper, .a8cBranchTest]
         case .unifiedSiteAddress:
-            return false
+            return true
```

Comment:
I don't think you meant to commit this... At least, I hope not. 😄

Reply:
oh no! Sorry about that, I had reviewed your PR just before and for some reason git allowed me to checkout my branch anyway... thanks for finding this!!